### PR TITLE
Added more models to Email.js

### PR DESCRIPTION
**Pull request by accakks**

### CODE REVIEW

File: models/doctype/Email/Email.js [hunk 1]
```diff
@@ -38,6 +38,38 @@ module.exports = {
...
+            "fieldname": "emailAddress",
+            "label": "cc",
+            "fieldtype": "Data",
+            "required": 1
+        },
+        {
+            "fieldname": "emailAddress",
+            "label": "bcc",
+            "fieldtype": "Data",
+            "required": 1
```
Comment (meets2tarun) [inline models/doctype/Email/Email.js]: In case if someone doesn't want to provide any `bcc` address.

File: models/doctype/Email/Email.js [hunk 2]
```diff
@@ -38,6 +38,38 @@ module.exports = {
...
+            "fieldname": "emailAddress",
+            "label": "to",
+            "fieldtype": "Data",
+            "required": 1
+        },
+        {
+            "fieldname": "emailAddress",
+            "label": "cc",
+            "fieldtype": "Data",
+            "required": 1
```
Comment (meets2tarun) [inline models/doctype/Email/Email.js]: If someone doesn't want to provide a `cc` address.

**Comment by accakks** [comment]
oops, I'll make changes to cc, bcc and make PR again!

**Comment by accakks** [comment]
Done @meets2tarun

File: models/doctype/Email/Email.js [hunk 3]
```diff
@@ -38,6 +38,38 @@ module.exports = {
...
+          "fieldname": "Subject",
+          "label": "Subject",
+          "fieldtype" : "Text",
+          "required": 0
+        },
+        { 
+          "fieldname": "Content",
+          "label": "",
+          "fieldtype" : "Text",
+          "required": 0
```
Comment (meets2tarun) [inline models/doctype/Email/Email.js:70]: I think this is body, so I'm not sure whether a mail can be send without a body.

Comment (accakks) [reply models/doctype/Email/Email.js:70]: Blank mails are allowed @meets2tarun . However, for blank subject, we might have to add a warning/prompt later "Send mail without any subject", but maybe we'll deal with that while adding interface

**Review by meets2tarun — APPROVED** [review]
LGTM ✈️

**Pull request merged**